`gulp local` broken

```
Error: Cannot find module '/home/andy/TypeScript/scripts/generateLocalizedDiagnosticMessages.js'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:453:25)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:201:16)
    at bootstrap_node.js:626:3
```

Probably missed something in #18702? @mhegazy 

Also, it looks like if I run `gulp local` *then* `gulp localize`, the second command will fail!

**EDIT**: Looks like `generateLocalizedDiagnosticMessages.js` is being saved to `/TypeScript` and not to `/TypeScript/scripts`.

--- a/Gulpfile.ts
+++ b/Gulpfile.ts
@@ -394,7 +394,7 @@ gulp.task(generateLocalizedDiagnosticMessagesJs, /*help*/ false, [], () => {
         .pipe(sourcemaps.init())
         .pipe(tsc(settings))
         .pipe(sourcemaps.write("."))
-        .pipe(gulp.dest("."));
+        .pipe(gulp.dest(scriptsDirectory));
 });
 
 // Localize diagnostics

